frontend/src/router.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+    BrowserRouter,
+    Routes,
+    Route,
+  } from "react-router-dom";
+import About from "./pages/About";
+
+import Home from "./pages/Home";
+import CanOfferShelter from "./pages/Signup/CanOfferShelter";
+import NeedShelter from "./pages/Signup/NeedShelter";
+
+export default function Router() {
+    return (
+        <BrowserRouter>
+            <Routes>
```

Comment:
I don't know much about React Router: I'm wondering could we just put the <Header> here so it shows up on every page?

Reply:
You mean Header component?